public/static/js/spawned_hosts.js

```diff
@@ -198,14 +179,17 @@ mciModule.controller('SpawnedHostsCtrl', ['$scope','$window', '$timeout', 'mciSp
     };
 
     $scope.updateHostExpiration = function() {
-      mciSpawnRestService.extendHostExpiration(
+        var new_expiration = new Date($scope.date_for_expiration);
```

Comment:
nit: declare variables with "let" rather than "var" so that they have predictable scoping behavior